sdk/workflow-tengo/src/smart.lib.tengo

```diff
@@ -699,13 +706,25 @@ resource = func(r) {
 		 * Gets value from key value storage associated with resource if corresponding
 		 * record exists.
 		 *
-		 * @param key string key
-		 * @return bytes or undefined
+		 * Note: it uses resourceKeyValueGetIfExists under the hood.
+		 *
+		 * @param key: string - key
+		 * @return bytes
 		 */
 		kvGet: func(key) {
 			return tx.resourceKeyValueGet(self.id, key)
 		},
 
+		/**
+		 * Gets a flag from key value storage associated with resource
+		 *
+		 * @param key: string - key
+		 * @return bool
+		 */
+		kvGetFlag: func(key) {
+			return tx.resourceKeyValueGetFlag(self.id, key)
```

Comment:
Let's do assertions with `feats...` here and in all other new methods. At least we'll get clear message on why we broke instead of 'indefined is not callable'